lib/std/os/bits/linux.zig

```diff
@@ -182,26 +184,40 @@ pub usingnamespace if (is_mips)
         pub const SA_NOCLDSTOP = 1;
         pub const SA_NOCLDWAIT = 0x10000;
         pub const SA_SIGINFO = 8;
+        pub const SA_RESTART = 0x10000000;
+        pub const SA_RESETHAND = 0x80000000;
 
         pub const SIG_BLOCK = 1;
         pub const SIG_UNBLOCK = 2;
         pub const SIG_SETMASK = 3;
     }
+else if (is_sparc)
+    struct {
```

Comment:
Ah, thanks for the correction!

> We should really stop with this copy-paste madness and write a small Zig script to autogenerate the constants list for all the platforms.

Pretty much this. While working on this I spent quite a lot of time just hunting/copy-pasting/fixing constants, because, among other reasons, Debian-supplied header files can be quite confusing, with the same constant defined multiple times with different values in different files...

Reply:
> Pretty much this. While working on this I spent quite a lot of time just hunting/copy-pasting/fixing constants, because, among other reasons, Debian-supplied header files can be quite confusing, with the same constant defined multiple times with different values in different files...

Yeah, and doing everything by hand is a great way to get copy/paste errors and such. Do you mind opening a ticket about this problem?

Reply:
Sure! I've opened a ticket [here](https://github.com/ziglang/zig/issues/6808).